### PR TITLE
Chinese parser for OSM

### PR DESCRIPTION
**Pull request by peitili**
https://github.com/tilezen/vector-datasource/issues/1955

- [x] Update tests
- [x] Update docs

### CODE REVIEW

File: vectordatasource/transform.py [hunk 1]
```diff
@@ -612,6 +623,42 @@ def _convert_osm_l10n_name(x):
     return LangResult(code=result, priority=priority)
 
 
+def post_process_osm_zh(properties):
+    """ First check whether name:zh (Simplified) and name:zht(Traditional)
```
Comment (travisgrigsby) [inline vectordatasource/transform.py:627]: great comment

**Review by travisgrigsby — APPROVED** [review]
nice - I'm assuming you'll add two more for the other two data sources in a followup.  Nice work with the tests exactly laying out how your code works

**Review by iandees — APPROVED** [review]

File: integration-test/1955-chinese-parser.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                                        u'is_in:continent': u'North America',
+                                        u'name:eu': u'San Francisco'}))  # noqa
+
+        self.assert_has_feature(
+            16, 10482, 25330, 'places',
+            {'id': 26819236, 'kind': 'locality', 'kind_detail': 'city',
+             'source': "openstreetmap.org",
+             'name': 'San Francisco',
+             'name:zh': '旧金山',
+             'name:zht': '舊金山'})
```
Comment (nvkelso) [inline integration-test/1955-chinese-parser.py:59]: Can you also add another assert for this same tile, same place ID that `name:zh-default` is NOT exported in tiles, please?

**Review by nvkelso — COMMENTED** [review]
One nit to address to make sure intermediate processing data is not exported into tiles (it's been an issue in the past we should always guard against).

**Comment by peitili** [comment]
> One nit to address to make sure intermediate processing data is not exported into tiles (it's been an issue in the past we should always guard against).

ok, I will remove the zh-default prop

**Review by nvkelso — CHANGES_REQUESTED** [review]
For the `docs` checkmark, thanks for updating the TESTS.md, but please also update 

https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md#name-localization

With a note about `name:zh` (Simplified Chinese characters as used in China and Singapore) and `name:zht` for (Traditional Chinese characters used in Taiwan, Hong Kong, Macau) and how the data value backfilling works, and that we don't auto-translate to simplified <> traditional.

**Comment by peitili** [comment]
> For the `docs` checkmark, thanks for updating the TESTS.md, but please also update
> 
> https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md#name-localization
> 
> With a note about `name:zh` (Simplified Chinese characters as used in China and Singapore) and `name:zht` for (Traditional Chinese characters used in Taiwan, Hong Kong, Macau) and how the data value backfilling works, and that we don't auto-translate to simplified <> traditional.


sure, do we also need to change CHANGELOG and VERSION stuffs or shall we wait until NatualEarth and WOF parser are done too?

**Comment by peitili** [comment]
> For the `docs` checkmark, thanks for updating the TESTS.md, but please also update
> 
> https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md#name-localization
> 
> With a note about `name:zh` (Simplified Chinese characters as used in China and Singapore) and `name:zht` for (Traditional Chinese characters used in Taiwan, Hong Kong, Macau) and how the data value backfilling works, and that we don't auto-translate to simplified <> traditional.

fixed

File: docs/layers.md [hunk 3]
```diff
@@ -52,6 +52,18 @@ For features in the `boundaries` layer, there are two additional variants `name:
 * `name:left:*` _See planned bug fix [#1102](https://github.com/tilezen/vector-datasource/issues/1102)._
 * `name:right:*` _See planned bug fix [#1102](https://github.com/tilezen/vector-datasource/issues/1102)._
 
+##### Chinese name localization
+Both `name:zh` and `name:zht` properties are best-effort. It means `name:zh` can sometimes contain Traditional Chinese and `name:zht` can sometimes contain Simplified Chinese. And we don't auto-translate Traditional Chinese to Simplified Chinese or vice versa.
```
Comment (nvkelso) [inline docs/layers.md:56]: Nit: add extra line between the header and the paragraph text to match the Markdown formatting used in this doc.

File: docs/layers.md [hunk 4]
```diff
@@ -52,6 +52,18 @@ For features in the `boundaries` layer, there are two additional variants `name:
...
+
+###### OSM
+We use three fields `name:zh`, `name:zh-Hans` and `name:zh-Hant` from OSM to populate two properties:
```
Comment (nvkelso) [inline docs/layers.md:59]: Nit: add extra line between the header and the paragraph text to match the Markdown formatting used in this doc.

File: integration-test/1955-chinese-parser.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+        self.assert_has_feature(
+            16, 10482, 25330, 'places',
+            {'id': 26819236, 'kind': 'locality', 'kind_detail': 'city',
+             'source': "openstreetmap.org",
+             'name': 'San Francisco',
+             'name:zh': u'旧金山',
+             'name:zht': u'舊金山'})
+
+        self.assert_no_matching_feature(16, 10482, 25330, 'places', {'name:zh-default': u'旧金山/三藩市/舊金山'})
```
Comment (nvkelso) [inline integration-test/1955-chinese-parser.py:61]: Nit: Please format this onto multiple lines like the other assert for legibility / convention.

**Review by nvkelso — APPROVED** [review]
Great inaugural PR, thank you!

The documentation changes work for me, though please address the 3 nits comments I left (all formatting) before merging the PR.

File: docs/TESTS.md [hunk 6]
```diff
@@ -1,5 +1,13 @@
 # Tilezen Vector Tile Tests
 
+## Setup
+In your virtual environment of this repo run the following commands to install dependencies
```
Comment (nvkelso) [inline docs/TESTS.md:4]: Nit: add another line here between header and paragraph text

**Pull request merged**